trunk/SUAVE/Analyses/Aerodynamics/Supersonic_Zero.py

```diff
@@ -62,11 +62,12 @@ def __defaults__(self):
         settings.trim_drag_correction_factor        = 1.02
         settings.wing_parasite_drag_form_factor     = 1.1
         settings.fuselage_parasite_drag_form_factor = 2.3
-        settings.aircraft_span_efficiency_factor    = 0.78
+        settings.aircraft_span_efficiency_factor    = 0.0
```

Comment:
Should this be None?

Reply:
No, it's a necessary parameter if you're using the supersonic zero analysis:
https://github.com/suavecode/SUAVE/blob/feature-PGM/trunk/SUAVE/Methods/Aerodynamics/Supersonic_Zero/Drag/induced_drag_aircraft.py#L53

Reply:
I could have it default to 0.78, but that's rather arbitrary.

Reply:
Wait, I take this all back. This parameter should be removed entirely.